### PR TITLE
Cleaned up dependency tables

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by Lex-2008** [comment]
looks good, but can you also make a follow-up PR to teach cf-bottom understand new tables?

**Comment by olehermanse** [comment]
> looks good, but can you also make a follow-up PR to teach cf-bottom understand new tables?

Nope. I mean, we can, but I don't think this is a high priority? Since I had to ping you to update these tables, it's not really automated anyway?

**Comment by olehermanse** [comment]
> looks good, but can you also make a follow-up PR to teach cf-bottom understand new tables?

@Lex-2008 Nope. I mean, we can, but I don't think this is a high priority? Since I had to ping you to update these tables, it's not really automated anyway?

**Comment by Lex-2008** [comment]
It's not automated in that sense that someone (me) needs to tell cf-bottom to do this. Or, rather, you have to ping me so I ping cf-bottom. :-D

File: deps-packaging/README.md [hunk 1]
```diff
@@ -2,18 +2,19 @@ File `install-dependencies` and the relevant subdirectories
...
+## Build dependencies
 
-* lcov
-* git
-* rsync
+| CFEngine version | 3.12.x | 3.15.x | master |
+| ---------------- | ------ | ------ | ------ |
+| lcov             | 1.14   | 1.14   | 1.14   |
+| git              |        |        |        |
+| rsync            |        |        |        |
```
Comment (vpodzime) [inline deps-packaging/README.md]: This is totally incomplete and wrong, I think.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+| rsync            |        |        |        |
+
+### Agent Dependencies
+
+| CFEngine version                                                                 | 3.12.x | 3.15.x | master | Notes                    |
+| -------------------------------------------------------------------------------- | ------ | ------ | ------ | ------------------------ |
+| [SASL2](https://cyrusimap.org/mediawiki/index.php/Downloads)                     | 2.1.27 | 2.1.27 | 2.1.27 | Solaris Enterprise agent |
+| [libacl](http://download.savannah.gnu.org/releases/acl/)                         | 2.2.53 | 2.2.53 | 2.2.53 |                          |
+| [libattr](http://download.savannah.gnu.org/releases/attr/)                       | 2.4.48 | 2.4.48 | 2.4.48 |                          |
+| [libcurl](http://curl.haxx.se/download.html)                                     | 7.70.0 | 7.70.0 | 7.68.0 |                          |
```
[the remark this answers is not in the document]
Comment (vpodzime) [reply README.md:89]: Yes, please! We should always have newer deps on *master* than any other branch.

**Pull request merged**